File: 351001/Budnikov/publisher/src/models/editor.py
from tortoise import fields, models


class Editor(models.Model):
    id = fields.IntField(pk=True)
    login = fields.CharField(max_length=64, unique=True)
    password = fields.CharField(max_length=128)
    firstname = fields.CharField(max_length=64)
    lastname = fields.CharField(max_length=64)
    role = fields.CharField(max_length=32, default="CUSTOMER")

    issues: fields.ReverseRelation["Issue"]

    class Meta:
        table = "tbl_editor"

File: 351002/Yanachkin/Lab1-5/lab1/Models/Entities/Editor.cs
namespace lab1.Models.Entities;

public class Editor : BaseEntity
{
    public string Login { get; set; } = null!;
    public string Password { get; set; } = null!;
    public string FirstName { get; set; } = null!;
    public string LastName { get; set; } = null!;

    public ICollection<Issue> Issues { get; set; } = new List<Issue>();
}
